package.json

```diff
@@ -100,10 +100,10 @@
     "ndarray-ops": "^1.2.2",
     "point-cluster": "^3.1.4",
     "polybooljs": "^1.2.0",
-    "regl": "^1.3.6",
+    "regl": "^1.3.7",
```

Comment:
only this https://github.com/regl-project/regl/pull/496 is new in `1.3.7` which only affects logging styles which should not have any effects for plotly.js